assets/js/components/modules-list.js

```diff
@@ -73,11 +72,17 @@ class ModulesList extends Component {
 	}
 
 	render() {
-		const modules = window.googlesitekit.modules || [];
+		// Filter out internal modules.
+		const modules = filter( window.googlesitekit.modules || [], ( module ) => ! module.internal );
```

Comment:
Consider using [`reject`](https://lodash.com/docs/4.17.14#reject) instead.
I.e. `reject( modules, ( module ) => module.internal )`

Reply:
I'd prefer to use a native array function instead, if we make these changes. `filter()` would work similarly to Lodash's one.